.github/workflows/pylint.yml

```diff
@@ -0,0 +1,23 @@
+name: Pylint
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
```

Comment:
Only the version that is used by OMV and Debian in the current release should be used here.